README.md

```diff
@@ -1,20 +1,88 @@
 # htbhf-acceptance-tests
-Acceptance tests for Apply for Healthy Start application
+Contains acceptance tests and compatibility tests for Apply for Healthy Start application using Cucumber and Selenium Webdriver to run Gherkin feature files. 
+Note that the compatibility test feature file (`/src/test/resources/features/compatibility/apply-success.feature`) is also run as a part of the acceptance
+tests to make sure that there is no error in this file before we run the compatibility tests.
 
-These tests are downloaded and run by the htbhf-applicant-web-ui project 
-(which specifies the version of these tests to run in `test_versions.properties`). 
-The acceptance tests are run as part of the CI build for htbhf-applicant-web-ui and the compatibility tests are run during the CD pipeline.
+## Building this project in Circle CI
+The build for this project by default downloads and uses the latest release of the htbhf-applicant-web-ui project to run the tests against. The 
+script `prepare_for_tests.sh` is used by Circle CI to download the appropriate version. The version used can be overridden by setting the environment variable
+`WEB_UI_BRANCH` which is useful when testing a specific branch of the web-ui. This can be done for the Circle CI build in the `.circleci/config.yml` file in 
+the `jobs/build/environment` section (NOTE: This value needs to be reset back to blank once the branch is deleted otherwise subsequent builds will fail. The
+build itself runs the following scripts in this order:
+
+ - `ci_scripts/prepare_for_tests.sh` - downloads and runs the appropriate version of web-ui and the session-details application
+ - `ci_scripts/download_chromedriver.sh` - downloads the appropriate version of chromedriver into the bin directory (see below for more specific details on this). 
```

Comment:
Should we say the project bin directory just so it's clear that it's not going to download things to /bin or /usr/bin, etc.

Reply:
Sure, I'll update.